ui/shared/components/panel/search/constants.js

```diff
@@ -357,8 +345,7 @@ export const SV_FREQUENCIES = [
   },
 ]
 
-export const FREQUENCIES = [...SNP_FREQUENCIES]
-FREQUENCIES.splice(5, 0, ...SV_FREQUENCIES)
+export const FREQUENCIES = [...SNP_FREQUENCIES, ...SV_FREQUENCIES]
```

Comment:
Besides the order, the ticket also asks to change the name of `SV Callset` to `This SV Callset`.

Reply:
oh good catch thank you!